### ISSUE TITLE
Clarification on GSM loss normalization

### ISSUE THREAD

**Issue by alessiasaporita** [Quanato607/MST-KDNet#4]
Hi, thank you for releasing the code and for the interesting work!

While studying the implementation of the GSM (Global Style Matching) loss, I noticed some ambiguity regarding the normalization strategy and wanted to ask for clarification.

Context

In the code, the mix/Gram matrix is computed as:
```python
input1 = input1.view(B * C, H * W * D)
input2 = input2.view(B * C, H * W * D)
G = torch.mm(input1, input2.t())
G = G / (B * C * H * W * D)
```

Then, in the loss, an additional normalization is applied:
```python
loss += ||M_f - M_m||^2 / (4 * C^2 * (H*W)^2)
```

I would like to better understand the intended normalization: 
Is the double normalization (inside mix_matrix and again in the loss via size^2) intentional?

Thanks in advance for your clarification!
